python/src/scipp/html/style.css.template

```diff
@@ -57,21 +74,23 @@
 }
 
 .sc-section-item input + label {
-  color: #ccc;
+  /*color: #ccc;*/
+  color: #0000aa;
 }
 
 .sc-section-item input:enabled + label {
   cursor: pointer;
-  color: #555;
+  color: var(--sc-font-color1);
 }
 
 .sc-section-item input:enabled + label:hover {
-  color: #000;
+  color: var(--sc-font-color0);
 }
 
 .sc-section-summary {
   grid-column: 1 ;
-  color: #555;
+  color: #00aaaa;
+  /*color: #555;*/
```

Comment:
commented code

Reply:
Ah right. I had manipulated those to figure out what they apply to. But I could not find anything. Do you know? Otherwise I just revert those three changes.

Reply:
No idea. They don't seem to be used anywhere.
Maybe you can just remove them?